File: Iniciante/1020 - Idade em Dias.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main()
{
    int dias, a, m, d;
    scanf("%d", &dias);

    a = floor(dias / 365);
    dias = floor(dias % 365);
    m = floor(dias / 30);
    d = floor(dias % 30);
    
    printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", a, m, d);

    return 0;
}

File: Iniciante/1020 - Idade em dias.py
from math import floor
d = int(input())

ano = floor(d / 365)
d = floor(d % 365)
m = floor(d / 30)
d = floor(d % 30)

print(f'{ano} ano(s)\n'
      f'{m} mes(es)\n'
      f'{d} dia(s)')